lazy_theta_star_b/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+cmake_minimum_required(VERSION 3.5)
+project(lazy_theta_star_b)
```

Comment:
What's the `_b` for? I'd think `lazy_theta_star_planner` would be a good name

Reply:
Was planning to change it in the final version, anyways. It has that _b because of it  being the package name in my computer, so as to be able to diffrentiate.

Reply:
OK, no worries on that right now